src/core/mavlink_parameters.cpp

```diff
@@ -21,13 +21,34 @@ MAVLinkParameters::MAVLinkParameters(SystemImpl& parent) : _parent(parent)
         MAVLINK_MSG_ID_PARAM_EXT_ACK,
         std::bind(&MAVLinkParameters::process_param_ext_ack, this, std::placeholders::_1),
         this);
+
+    // Parameter Server Callbacks
+    _parent.register_mavlink_message_handler(
+        MAVLINK_MSG_ID_PARAM_REQUEST_READ,
+        std::bind(&MAVLinkParameters::process_param_request_read, this, std::placeholders::_1),
```

Comment:
The idea is to use lambdas instead of `std_bind` moving forward. However, we can also change that in a PR later.